src/Robo/Tasks/Drush.php

```diff
@@ -0,0 +1,267 @@
+   *
+   * @param string $command
+   *
+   * @return $this
+   */
+  public function drush($command) {
+    // @todo Figure out how to fetch config from constructor to avoid this.
+    if (!$this->defaultsInitialized) {
+      $this->init();
+    }
+
+    if ($this->alias) {
+      $command = "@{$this->alias} {$command}";
+    }
+
+    if (!empty($this->uri)) {
+      $this->option('-l', $this->uri);
+    }
+
+    if (isset($this->assume) && is_bool($this->assume)) {
+      $assumption = $this->assume ? 'yes' : 'no';
+      $this->option("--{$assumption}");
+    }
+
+    if ($this->verbosityThreshold() >= VerbosityThresholdInterface::VERBOSITY_VERBOSE) {
+      $this->verbose(TRUE);
+    }
+
+    if ($this->verbose) {
+      $this->option('-v');
+    }
+
+    // Add in arguments set via option method and clear for next invocation.
+    $command = $command . $this->arguments;
+    $this->arguments = '';
+
+    return $this->exec($command)
+      ->dir($this->dir)
+      ->interactive($this->passthru)
+      ->printOutput($this->logOutput)
+      ->printMetadata(false);
+  }
+
+  /**
+   * Sets the site alias to be used for each command.
+   *
+   * @param string $alias
+   *
+   * @return $this
+   */
+  public function alias($alias) {
+    $this->alias = $alias;
+    return $this;
+  }
+
+  /**
+   * Sets the site uri to be used for each command.
+   *
+   * @param string $uri
+   *
+   * @return $this
+   */
+  public function uri($uri) {
+    $this->uri = $uri;
+    return $this;
+  }
+
+  /**
+   * Sets the working directory for each command.
+   *
+   * @param string $dir
+   *
+   * @return $this
+   *
+   * @see ExecTrait::$workingDirectory
+   */
+  public function dir($dir) {
+    $this->dir = $dir;
+    parent::dir($dir);
+    return $this;
+  }
+
+  /**
+   * Assume 'yes' or 'no' to all prompts.
+   *
+   * @param string|bool $assume
+   *
+   * @return $this
+   */
+  public function assume($assume) {
+    if ($assume === "") {
+      $this->assume = '';
+    }
+    elseif (is_string($assume)) {
+      $this->assume = ($assume === 'yes' || $assume === 'true');
+    } else {
+      $this->assume = !!$assume;
+    }
+    return $this;
+  }
+
+  /**
+   * Attach tty to process for interactive input.
+   *
+   * @param string|bool $passthru
+   *
+   * @return $this
+   *
+   * @see ExecTrait::$interactive
+   */
+  public function passThru($passthru) {
+    if (is_string($passthru)) {
+      $this->passthru = ($passthru === 'yes' || $passthru === 'true');
+    } else {
+      $this->passthru = !!$passthru;
+    }
+    return $this;
+  }
+
+  /**
+   * Indicates if command output should be printed.
+   *
+   * @param string|bool $logOutput
+   *
+   * @return $this
+   *
+   * @see ExecTrait::$isPrinted
+   */
+  public function logOutput($logOutput) {
+    if (is_string($logOutput)) {
+      $this->logOutput = ($logOutput === 'yes' || $logOutput === 'true');
+    } else {
+      $this->logOutput = !!$logOutput;
```

Comment:
Agreed, lol - I borrowed this notation [from the existing `DrushTask`](https://github.com/acquia/blt/blob/8.x/phing/phingcludes/DrushTask.php#L223) from the `phing` days. Happy to change.